playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -98,6 +98,17 @@
   tasks:
   - set_fact:
       openshift_rpm: "{{ hostvars[groups['nodes'][0]]['yum'].results | selectattr('name', 'match', '^(origin|atomic-openshift)$') | first }}"
+  - name: create a file with image name
+    copy:
+      content: |
+        ---
+        image_name: {{ openshift_azure_output_image_name }}
+        base_image: {{ (input_image.stdout | from_json).name }}
```

Comment:
should we add the resource group for good measure (openshift_azure_input_image_ns)?